docs/manual/manual_velocity_control_mechanisms.rst

```diff
@@ -0,0 +1,419 @@
+
+.. _tuning_parameters:
+
+Tuning Velocity Gains and Parameters
++++++++++++++++++++++++++++++++++++++++
+Propeller Motor Control PID Tuning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+As mentioned :ref:`above <control_with_prop_motor_control>`, tuning the Propeller Motor Controller and Multi Turn Angle Controller differs due 
+to each controller's nature. For more information about tuning the Multi Turn Angle Controller please see :ref:`Position Control Tuning <position_control_tuning>`.
+
+Our velocity PID controller found in :ref:`Propeller Motor Controller <propeller_motor_control>` can be tuned as any standard PID controller. 
+With this controller, the proportional term acts on a velocity, our integral term on position, and our derivative term on acceleration. 
+
+Velocity Kp
+----------------
+The velocity proportional gain. Helps to correct speed errors. Higher gain means smaller errors and faster response, but can lead to oscillations and overheating.
+
+Velocity Ki
+----------------
+The velocity integral gain. Helps to correct steady state speed errors. Higher gain means quicker suppression of steady state errors, but also increased oscillations, overheating, and faster windup. This term is generally not needed.
+
+Velocity Kd
+----------------
+The velocity derivative gain. Helps to reduce overshoot oscillations. Higher gain means less overshoot from Kp and Ki, but can also introduce higher frequency oscillations and overheating.
+
+Velocity Feed Forward Tuning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Feed forward terms are those added to the calculated PID control correction, and are based on the reference velocity. 
+There are 3 available feed forward parameters settable through the Propeller Motor Controller, Feed Forward 0, 1, and 2. 
+Feed Forward 0 is a constant value, Feed Forward 1 is multiplied by the current reference velocity, and Feed Forward 2 
+is multiplied by the reference velocity squared.
+
+.. figure:: ../_static/manual_images/velo_voltage_control/feed_forward_terms.png
+    :align: center
+    :scale: 90%
+    :alt: Feed Forward Terms in PID Control
+
+    Feed Forward Terms in Velocity PID Control
+
+Velocity Feed Forward 0
+--------------------------
+The 0th order feedforward for the velocity controller. The constant voltage offset. Generally not needed for propellers. Velocity FF0 has units of volts [:math:`V`].
+
+Velocity Feed Forward 1
+--------------------------
+The 1st order feedforward for the velocity controller. A linear voltage term, which compensates for friction and motor back EMF. We recommend keeping the default value. Velocity FF1 has units of volts per radian per second [:math:`\frac{V}{(\frac{rad}{s})}`].
+
+Velocity Feed Forward 2
+--------------------------
+The 2nd order feedforward of the velocity controller. Typically compensates for propeller drag. Velocity FF2 has units of volts per radian per second squared [:math:`\frac{V}{(\frac{rad}{s^2})}`].
+
+Full Propeller Motor Controller
```

Comment:
We talked about it in person, just putting it here for record keeping: Would be good to show summary images of the two full controllers at the beginning so there is a clear place to see the full controller and to help understand that this documentation is going to walk you through building it up one piece at a time.